decidim/decidim
# Fix for saving an Election that wasn't blocked

#### :tophat: What? Why?

While working in `decidim-elections`, I noticed that if you have an Election with an `start_date in the past (ie an ongoing Election that wasn't sent to the Bulletin Board), the application doesn't allow you to make some changes in it (like creating/deleting/updating Questions and Answers).

This doesn't make sense, as it shouldn't allow admins to make changes only once the Election was sent to the Bulletin Board, as making changes once the Election was sent could generate synchronization issues (like adding a question after it was sent). For signaling this we don't use the `start_date` attribute, but the `blocked_at` attribute, accessible through the `election#blocked?` method. 

This PR fixes that. 

#### Testing

(Before the patch)

1. Sign in as admin and go to the admin panel (/admin)
2. Go to an Election inside of a Participatory Space (ie an Assembly) 
3. Change the `start_date` of an Election to some time in the past (like 1 week ago)
4. Try to update a Question or an Answer
5. See that's not allowed  

(After the patch)

1. Sign in as admin and go to the admin panel (/admin)
2. Go to an Election inside of a Participatory Space (ie an Assembly) 
3. Change the `start_date` of an Election to some time in the past (like 1 week ago)
4. Try to update a Question or an Answer
5. See that's allowed 

:hearts: Thank you!
